js/views/chatview.js

```diff
@@ -82,8 +80,6 @@
 		},
 
 		initialize: function(options) {
```

Comment:
Eslint fails:

> /drone/src/github.com/nextcloud/spreed/js/views/chatview.js
82:24 error 'options' is defined but never used no-unused-vars

Reply:
Fixed.